examples/startup_search/encoder.py

```diff
@@ -0,0 +1,49 @@
+from typing import Any, Dict, List, Union
+
+from quaterion_models.encoders import Encoder
+from quaterion_models.types import CollateFnType
+
+try:
+    from sentence_transformers import SentenceTransformer
+except ImportError:
+    import sys
+
+    print("You need to install sentence-transformers for this example")
+    sys.exit(1)
+
+
+class StartupEncoder(Encoder):
+    def __init__(self, encoder_path: str):
```

Comment:
why did it become `encoder_path` instead of `pretrained_name`?

Reply:
I think it's more self-explanatory as it's also used when restoring the fine-tuned model with `.load()`